js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "bitcoin-spv-js",
+    "author": "Erin Hales, Dominique Liau, James Prestwich",
```

Comment:
More of a nit to help out automation around package management and introspection, the `author` field here doesn't fit the schema defined by npm - https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Reply:
fixed